src/www/index.html

```diff
@@ -315,5 +311,20 @@
     <span> <a id="deploy-link"> </a> </span>
   </div>
  </div>
+  <script>
```

Comment:
@abishekvashok Check this 
```
    $(document).on('mouseover', function () {
        /* Clearing any timeout set under browserSuggestionTweak */
        $('.overlay')[0].style.opacity = 1;
    });
    $(document).on('mouseout', function () {
        /* Set a timeout as immediatly displaying the overlay when suggestions
        appear in the browser will be very bad.*/
        $('.overlay')[0].style.opacity = "";
    })
```

Reply:
It will still produce the bug @geekyd 
I tried that

Reply:
@abishekvashok This problem exists for firefox. In chrome It flickers instead of the opacity=1;
If the user is selecting the emails for more than 3 seconds then the problem will be same. What do you think about it? How should we take care of that?